src/metabase/public_settings.clj

```diff
@@ -942,3 +942,10 @@ See [fonts](../configuring-metabase/fonts.md).")
   :export?    false
   :default    true
   :type       :boolean)
+
+(defsetting download-row-limit
+  (deferred-tru "Exports row limit, not including xlsx.")
```

Comment:
I'm just thinking how this could be misinterpreted: this is the row limit not including the header. But someone could think this is the number of rows in the CSV, including the header.

Also, this is currently a bit misleading because the `download-row-limit` will be binding if it is less than the `absolute-max-results` for an xlsx download.

Something like this might be more accurate:
```suggestion
  (deferred-tru "Exports row limit excluding the header. xlsx downloads are limited to 1048575 rows even if this limit is higher.")
```